website/public/main.css

```diff
@@ -718,7 +718,73 @@ kbd.keystroke {
     bottom: 0;
   }
 }
+.show-in-mobile {
+  display: none;
+}
+@media only screen and (max-width: 43em) {
+  /* Hide the feature, JShint and Deno columns */
+  .comparison tr th:first-child,
+  .comparison table th:nth-child(4),
+  .comparison table th:nth-child(6),
+  .comparison table th:nth-child(7),
+  .comparison table td:nth-child(4),
+  .comparison table td:nth-child(6),
+  .comparison table td:nth-child(7),
```

Comment:
Hmm, there's got to be a better way...

`col.whatever { visibility: collapse; }` works in Blink but not Gecko. =\ From MDN:

> Firefox doesn't hide borders when hiding [<col>](https://developer.mozilla.org/docs/Web/HTML/Element/col) and [<colgroup>](https://developer.mozilla.org/docs/Web/HTML/Element/colgroup) elements if border-collapse: collapse is set.
https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

I think we want `border-collapse: collapse;`, so I guess we are stuck with either your solution to adding a class to each `th` and `td`. =\